Client/Cards/create/ixaris.spec.ts

```diff
@@ -0,0 +1,45 @@
+import * as dotenv from "dotenv"
+import * as pax2pay from "../../../index"
+import { ErrorResponse } from "../../../model"
+import { factory } from "./factory"
+
+dotenv.config()
+jest.setTimeout(200000)
+
+describe("pax2pay.cards.create ixaris", () => {
+	const client = pax2pay.Client.create(process.env.url)
+	for (const currency of ["EUR", "GBP", "USD"])
+		for (const cardType of ["VISA_DEBIT", "VISA_CREDIT", "MASTERCARD"])
+			it(`card ${currency} ${cardType}`, async () => {
+				//authing inside test to reach trackingId for unique friendly names, maybe there's another way
```

Comment:
i dont understand why you're doing this? the combination of "cardType + currency," is going to be unique for each one regardless.